idna/src/uts46.rs

```diff
@@ -279,11 +279,11 @@ fn passes_bidi(label: &str, is_bidi_domain: bool) -> bool {
 /// V1 (NFC) and V8 (Bidi) are checked inside `processing()` to prevent doing duplicate work.
 ///
 /// http://www.unicode.org/reports/tr46/#Validity_Criteria
-fn is_valid(label: &str, config: Config) -> bool {
+fn is_valid(label: &str, config: Config, errors: &mut Errors) {
```

Comment:
since we're not returning anything, maybe we should call it `check_validity`?

Reply:
Fair enough, fixed that.